apps/AEPSampleAppNewArchEnabled/app/_layout.tsx

```diff
@@ -4,10 +4,22 @@ import 'react-native-reanimated';
 import { Drawer } from 'expo-router/drawer';
 
 import { useColorScheme } from '@/hooks/useColorScheme';
+import { MobileCore } from '@adobe/react-native-aepcore';
 
 // Prevent the splash screen from auto-hiding before asset loading is complete.
 SplashScreen.preventAutoHideAsync();
+const initOptions = {
```

Comment:
By default, use the `MobileCore.initializeWithAppID()` API.
Comment out the current logic under "If you need more customization, you can use InitOptions."